### PR TITLE
Migrating the .NET Core version to version 5.0

### PR DESCRIPTION
**Pull request by joaovictordecastroatake**
### Migrating the template for use the version 5.0 of ASP.NET Core Framework.

The migration is based on a article of Microsoft found [here](https://docs.microsoft.com/pt-br/aspnet/core/migration/31-to-50?view=aspnetcore-5.0&tabs=visual-studio).

#### Prerequisites

.NET SDK 5.0 or later

### CODE REVIEW

File: content/Api/Blip.Api.Template/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine3.11 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:5.0-alpine3.11 AS base
```
Comment (brandaogabriel7) [inline content/Api/Blip.Api.Template/Dockerfile:1]: This alpine image doesn't exist. Check the [Docker hub page](https://hub.docker.com/_/microsoft-dotnet-sdk/) to find one that suits. I would suggest using the 5.0-alpine3.12 version.

File: content/Api/Blip.Api.Template/Dockerfile [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 WORKDIR /app
 EXPOSE 80
 
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1-alpine3.11 AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:5.0-alpine3.11 AS build
```
Comment (brandaogabriel7) [inline content/Api/Blip.Api.Template/Dockerfile:5]: Same about the docker image here.

File: content/Api/Blip.Api.Template/Dockerfile (hunk 1 above)
Comment (brandaogabriel7) [inline content/Api/Blip.Api.Template/Dockerfile:1]: This alpine image doesn't exist. Check the [Docker hub page](https://hub.docker.com/_/microsoft-dotnet-sdk/) to find one that suits. I would suggest using the 5.0-alpine3.12 version.

File: content/Api/Blip.Api.Template/Blip - Backup.Api.Template.csproj [hunk 3]
```diff
@@ -0,0 +1,76 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```
Comment (brandaogabriel7) [inline content/Api/Blip.Api.Template/Blip - Backup.Api.Template.csproj:1]: I don't think you need this backup file here.

**Review by brandaogabriel7 — CHANGES_REQUESTED** [review]

File: content/Api/Blip.Api.Template/Dockerfile (hunk 2 above)
Comment (joaovictordecastroatake) [reply content/Api/Blip.Api.Template/Dockerfile:5]: In this case, mcr.microsoft.com/dotnet/core/sdk:5.0 has moved to dotnet/sdk. I will upgrade this line of code.

File: content/Api/Blip.Api.Template/Dockerfile (hunk 1 above)
Comment (joaovictordecastroatake) [reply content/Api/Blip.Api.Template/Dockerfile:1]: yeah, its true. In this case we have available this tags: 

5.0.103-alpine3.13-amd64, 5.0-alpine3.13-amd64, 5.0-alpine-amd64, 5.0.103-alpine3.13, 5.0-alpine3.13, 5.0-alpine

I will use 5.0-alpine3.13

**Review by brandaogabriel7 — COMMENTED** [review]
After @joaovictordecastroatake and I talked to our SRE (@luizjoli), it was decided that this template won't be updated for now as the .NET 5 images are not stable just yet. This will be done soon.

**Pull request closed without merging**